.npmignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+LICENCE
+dist/**/*.map
```

Comment:
There are a lot of useless file packaged in NPM. please ignore these folders:
```
src/**/*
test/**/*
build/**/*
libs/**/*
```

(add `!src/**/*.ts` later if we use typescript)